doc/source/index.rst

```diff
@@ -12,18 +12,12 @@ Qibolab is the dedicated `Qibo <https://github.com/qiboteam/qibo>`_ backend for
 quantum hardware control. This module automates the implementation of quantum
 circuits on quantum hardware. Qibolab includes:
 
-#. :ref:`Platform API <main_doc_platform>`: support custom allocation of quantum hardware platforms / lab setup.
+#. :ref:`Platform API <main_doc_platform>`: support custom allocation of quantum hardware platforms and lab setup.
+#. :ref:`Pulse API <main_doc_pulses>`: provide a library of custom pulses for execution through instruments.
 #. :ref:`Drivers <main_doc_instruments>`: supports commercial and open-source firmware for hardware control.
```

Comment:
Indeed, but this looked more like a summary of what qibolab provides in general, rather than how it is supposed to be used, so I left it as it is, just moving the "pulse API" higher, since this and the platform are the major points. If you see the next bullets it is even talking about the compiler (not even part of the public API) and circuit execution, which are not the main points of qibolab itself, but are still there in terms of code.

Feel free to change something if you like though.